src/content/structured/components/cards/code.mdx

```diff
@@ -259,9 +353,15 @@ export const adornment = [
   heading="Card heading" 
   subheading="Card subheading" 
   message="This is an example of a card with adornment.">
-  <SlottedSVG slot="icon" ... >
-    ...
-  </SlottedSVG>
+    <SlottedSVG slot="icon"  xmlns="http://www.w3.org/2000/svg"
```

Comment:
The gap between slot="icon" and xmlns seems to be bigger than a space, could we move xmlns to a new line?